troubleshoot/index.md

```diff
@@ -57,14 +57,44 @@ If you have an [Elastic subscription](https://www.elastic.co/pricing), you can c
   If you contact us by email, use the email address you registered with so we can help you more quickly. If your registered email is a distribution list, you can register a second email address with us. Just open a case to let us know the name and email address you want to add.
   :::
 
+  :::{warning}
+  All cases opened by email are opened at normal severity. For incidents, kindly ensure to raise via the [Elastic Support Portal](https://support.elastic.co/) at the [appropriate severity](https://www.elastic.co/support/welcome#what-to-say-in-a-case). 
```

Comment:
What about this? Generally you want to avoid terms like "kindly" as a direct instruction is shorter and clearer.

```suggestion
  All cases opened by email default to normal severity. For incidents, open a case through the [Elastic Support Portal](https://support.elastic.co/) and select the [appropriate severity](https://www.elastic.co/support/welcome#what-to-say-in-a-case). 
```